Healthy/Classes/ReusableViews/FormTextField/FormTextField.swift

```diff
@@ -0,0 +1,90 @@
+import UIKit
+
+class FormTextField: UIView {
+    
+    // MARK: Outlets
+    
+    @IBOutlet private weak var contentView: UIView!
+    @IBOutlet private weak var titleLabel: UILabel!
+    @IBOutlet private weak var inputTextField: UITextField!
+    @IBOutlet private weak var errorLabel: UILabel!
+    
+    // MARK: Properties
+    
+    private var onChange: (String) -> () =  { _ in }
+    
+    var title: String = "" {
+        didSet {
+            self.titleLabel.text = title
+        }
+    }
+    
+    var placeholder: String = "" {
+        didSet {
+            self.inputTextField.placeholder = placeholder
+        }
+    }
+    
+    var keyboardType: UIKeyboardType = .default {
+        didSet {
+            self.inputTextField.keyboardType = keyboardType
+        }
+    }
+    
+    var error: String = "" {
+        didSet {
+            self.errorLabel.text = error
+        }
+    }
```

Comment:
```suggestion
    var title: String = "" {
        didSet {
            self.titleLabel.text = title
        }
    }
    
    var placeholder: String = "" {
        didSet {
            self.inputTextField.placeholder = placeholder
        }
    }
    
    var keyboardType: UIKeyboardType = .default {
        didSet {
            self.inputTextField.keyboardType = keyboardType
        }
    }
    
    var error: String = "" {
        didSet {
            self.errorLabel.text = error
        }
    }
```

- All of these should be moved to a separate extension.
- They can act as setters/getters only without having any value to have only on source of truth (The views)